### DISCUSSION TITLE
Create Multibot for different channel

### DISCUSSION THREAD

**Discussion by HomaEEE** [def-studio/telegraph#300] [Q&A]
Hello, how  I can set settings two and more bots for many channels/chats with different info and functions?

**Comment by fabio-ivona** [comment]
Sorry could you elaborate your needs

**Comment by HomaEEE** [comment]
for example: 
Bot1 for personal - make order in pos system
Bot2 for client - notification and booking 

bots has different permission for database access and commands set

**Comment by fabio-ivona** [comment]
What prevents you from doing this?
